src/Turbo/composer.json

```diff
@@ -60,7 +60,8 @@
         "dbrekelmans/bdi": "dev-main"
     },
     "conflict": {
-        "symfony/flex": "<1.13"
+        "symfony/flex": "<1.13",
+        "symfony/web-link": "<5.4.0"
```

Comment:
This fix failing tests on lowest deps, it installed symfony/web-link `4.4.9` and failed because `AddLinkHeaderListener` tried to use `ResponseEvent::isMasterRequest()` (instead of `isMainRequest()`).

Reply:
Aaah no, that's because I removed the `env: SYMFONY_REQUIRE: '>=5.4'` in #2734